May 12, 2020 2:39:59 PM: Exception thrown: Unable to connect to item database
se.kth.iv1350.salesystem.controller.FatalSystemOperationException: Unable to connect to item database
	at se.kth.iv1350.salesystem.controller.Controller.internalAddItem(Controller.java:120)
	at se.kth.iv1350.salesystem.controller.Controller.addItemToSale(Controller.java:87)
	at se.kth.iv1350.salesystem.view.View.fakeInteractiveSale(View.java:96)
	at se.kth.iv1350.salesystem.startup.Main.main(Main.java:27)
Caused by: se.kth.iv1350.salesystem.integration.InventoryDBException: Unable to connect to external inventory system
	at se.kth.iv1350.salesystem.integration.InventorySystemHandler.getItemData(InventorySystemHandler.java:54)
	at se.kth.iv1350.salesystem.integration.ExternalDBHandler.getItemData(ExternalDBHandler.java:46)
	at se.kth.iv1350.salesystem.controller.Controller.internalAddItem(Controller.java:111)
	... 3 more
